_posts/2017-12-05-book-review-docker-up-running.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Book review 'Docker Up & Running'"
+date: "2017-12-05 11:26:32 +0530"
+tags:
+  - books
+  - docker
+  - linux
+---
+
+In the modern era of software engineering, terms are coined with a new wrapper.
+Such wrappers are required to make bread-and-butter out of it. Sometimes good
+marketed terms are adopted as best practices. I was having lot of confusions
+about this Docker technology. Even I was unfamiliar with the concept of
+containers. My certain goal was to get a higher level overview first and then
+come to a conclusion. I started reading about the Docker from its official
+getting started guide. That guide helped me to deploy this blog using Docker,
+but rather than solving initial it created few more questions. I decided to look
+for a better resource. By observing some Quora posts and Goodreads reviews, I
+decided to read “Docker Up & Running by K. Matthias & S. Kane”. I am sharing my
+reading experience here.
+
+TL;DR
+
+The book provides a nice overview of Docker toolchain. It is not a reference
+book. Even though few options are deprecated, I will advise you to read this
+book and then refer to the official documentation to get aware with the latest
+development.
+
+
+Detailed overview
+
+I got a printed copy at nearly 450 INR (roughly rounding to 7 USD, where 1 USD =
+65 INR) from Amazon. That prize is fairly acceptable with respect to the print
+quality. The book begins with a little history of the container (Docker is an
+implementation of the container). Initial chapters gives a higher level overview
+of Docker tools combining Docker engine, Docker image, Docker registry, Docker
+compose and Docker container. Authors had indicated software patterns where
```

Comment:
I like this suggestion particularly because it signifies that "software pattern" is a fancy term for repeating situations (just like ultimatecoder points out in the first paragraph for Docker).